invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/menu.overrides

```diff
@@ -57,13 +57,18 @@
     padding-right: 15px;
   }
 
-  .user-profile {
-    a.ui.button {
-      width: 13em;
+  #user-profile-dropdown {
+    #user-profile-dropdown-btn {
+      width: 13rem;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
     }
+
+    #user-profile-menu.ui.menu {
+      margin-left: .25rem;
+      width: 13rem;
```

Comment:
why is the width needed? wouldn't it be better to move it to the parent #user-profile-dropdown instead of applying twice the same rule for the children ?